sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/python/UserDefinedPythonDataSource.scala

```diff
@@ -172,7 +172,8 @@ case class UserDefinedPythonDataSource(dataSourceCls: PythonFunction) {
       pythonRunnerConf,
       metrics,
       jobArtifactUUID,
-      sessionUUID)
+      sessionUUID,
+      None)
```

Comment:
Can we also enable it here? IIUC, this will run on executors, so it would help to debug python datasources. cc @allisonwang-db

Reply:
Yep added it and a simple test for it